src/types/fixedtimezone.jl

```diff
@@ -1,3 +1,7 @@
+# Ideally would always use ShortString15, but it's `hash` is broken on 32-bit systems.
+# https://github.com/JuliaString/MurmurHash3.jl/issues/12
+const FixedTimeZoneName = Int === Int64 ? ShortString15 : String
```

Comment:
I really wish the 32-bit problem was properly fixed... if this is what we need to do then let's be as clear about it as we can.


```suggestion
const FixedTimeZoneName = Sys.WORD_SIZE == 64 ? ShortString15 : String
```

Reply:
It is now fixed